src/Raven.Client/Documents/Session/InMemoryDocumentSessionOperations.cs

```diff
@@ -58,6 +58,8 @@ public abstract partial class InMemoryDocumentSessionOperations : IDisposable
 
         private BatchOptions _saveChangesOptions;
 
+        public bool DisableAtomicDocumentWritesInClusterWideTransaction;
```

Comment:
internal readonly?

Reply:
No, this is so the user can disable that for a single session.